cmd/autoowners/main.go

```diff
@@ -0,0 +1,441 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+
+	"k8s.io/test-infra/experiment/autobumper/bumper"
+	"k8s.io/test-infra/prow/config/secret"
+	"k8s.io/test-infra/prow/github"
+)
+
+const (
+	doNotEdit     = "# DO NOT EDIT; this file is auto-generated using tools/populate-owners.\n"
+	ownersComment = "# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md\n"
+	//ownersAliasesComment = "# See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#owners_aliases\n"
+
+	githubOrg  = "openshift"
+	githubRepo = "release"
+)
+
+// owners is copied from k8s.io/test-infra/prow/repoowners's Config
```

Comment:
Since we have this in vendor can we just use it on this one?

Reply:
It was actually added the `yaml` tags.